docs/tests.md

```diff
@@ -84,7 +84,17 @@ testing portions of your code that can be tested without relying on a multiworld
 
 ## Running Tests
 
-In PyCharm, running all tests can be done by right-clicking the root `test` directory and selecting `run Python tests`.
-If you do not have pytest installed, you may get import failures. To solve this, edit the run configuration, and set the
-working directory of the run to the Archipelago directory. If you only want to run your world's defined tests, repeat
-the steps for the test directory within your world.
+#### Using Pycharm
+In PyCharm, running all tests can be done by right-clicking the root test directory and selecting Run 'Archipelago Unittests'. 
+If you do not have pytest installed, you may get import failures. To solve this, edit the run configuration, 
+and set the working directory to the Archipelago directory which contains all the project files. 
+
+If you only want to run your world's defined tests, repeat the steps for the test directory within your world.
+Your working directory should be the directory of your world in the worlds directory and the script should be the 
+tests folder within your world.
+
+If configured, you can also find the 'Archipelago Unittests' as an option in the dropdown at the top of the window
+next to the run and debug buttons.
```

Comment:
One point of the ./.run/* file is that you don't need to configure it, but it's already there. (This also applies to the first sentence in "Using Pycharm").

This is taken from a fresh clone:

![run-tests](https://github.com/user-attachments/assets/927a5254-cb92-4930-832e-29f3e7149e3a)